Analysis/Stan/mapping_model_inference.stan

```diff
@@ -442,10 +526,10 @@ model {
           lpmf = poisson_lpmf(y[ind_right[i]] | modeled_cases[ind_right[i]]);
         } else if (obs_model == 2) {
           // Quasi-poisson likelihood
-          lpmf = neg_binomial_2_lpmf(y[ind_right[i]] | modeled_cases[ind_right[i]], od_param * modeled_cases[ind_right[i]]);
+          lpmf = neg_binomial_2_lpmf(y[ind_right[i]] | modeled_cases[ind_right[i]], od_param[map_obs_admin_lev[ind_right[i]]] * modeled_cases[ind_right[i]]);
```

Comment:
random question but why do we need to index ind_right by i here but not when building the likelihood from the full observations ?

Reply:
For the full observations we can use the vectorized version of the likelihood functions by passing directly the vector of full indices, for the censored observations we cannot do this since we have (for now) to do individual computations to evaluate whether the censoring likelihood makes sense or not to avoid numerical errors (this is the "heuristic" mentioned in the comment"). 

In general, if we can we should use vectorized calls for efficiency, but I am not sure we can always afford to do this.